### PR TITLE
Update PPP and fix error message

### PR DESCRIPTION
**Pull request by inas0ng**
This commit
- updates my individual PPP with draft content
- fixes DateTime constraint error message, which may be misleading for users who interpret "valid range" as a "valid date value"

### CODE REVIEW

**Review by weilin1202 — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/model/event/DateTime.java [hunk 1]
```diff
@@ -12,7 +12,8 @@
  */
 public class DateTime {
     public static final String MESSAGE_CONSTRAINTS =
-            "Date times should follow this format: DD-MM-YYYY HH:mm and only values within valid range are allowed.";
+            "Date times should follow this format: DD-MM-YYYY HH:mm and "
+            + "only values within valid range are allowed (start date time cannot be after the end date time).";
```
Comment (venuslimm) [inline src/main/java/seedu/address/model/event/DateTime.java:16]: Will this affect the tests?

**Review by venuslimm — APPROVED** [review]
I left a comment 😄 but other than that, lgtm!

File: src/main/java/seedu/address/model/event/DateTime.java (hunk 1 above)
Comment (inas0ng) [reply src/main/java/seedu/address/model/event/DateTime.java:16]: nope! the tests call `DateTime.MESSAGE_CONSTRAINTS` directly, so no tests are affected by this change :-)

**Pull request merged**